### PR TITLE
optimize consensus nodes sync block

### PR DESCRIPTION
**Pull request by xiemylogos**

### CODE REVIEW

File: consensus/vbft/chain_store.go [hunk 1]
```diff
@@ -132,6 +134,7 @@ func (self *ChainStore) AddBlock(block *Block) error {
 		log.Errorf("chainstore AddBlock GetBlockExecResult: %s", err)
 		return fmt.Errorf("chainstore AddBlock GetBlockExecResult: %s", err)
 	}
+	log.Infof("chainstore addblock pendingBlocks height:%d,block height:%d", blkNum, block.getBlockNum())
```
Comment (Honglei-Cong) [inline consensus/vbft/chain_store.go]: remove or change to debug log

File: consensus/vbft/node_sync.go [hunk 2]
```diff
@@ -403,24 +403,25 @@ func (self *PeerSyncer) run() {
...
-			m, ok := p.(*blockProposalMsg)
-			if !ok {
-				panic("")
-			}
-			if m.Block.getProposer() == blkProposers[blkNum] {
-				proposalBlock = m.Block
-				break
+		proposalBlock, _ = self.server.blockPool.getSealedBlock(blkNum)
+		/*
+			for _, p := range self.server.msgPool.GetProposalMsgs(blkNum) {
```
Comment (Honglei-Cong) [inline consensus/vbft/node_sync.go]: remove unused code

File: consensus/vbft/service.go [hunk 3]
```diff
@@ -33,7 +33,7 @@ import (
 	"github.com/ontio/ontology/common"
 	"github.com/ontio/ontology/common/log"
 	actorTypes "github.com/ontio/ontology/consensus/actor"
-	"github.com/ontio/ontology/consensus/vbft/config"
+	vconfig "github.com/ontio/ontology/consensus/vbft/config"
```
Comment (Honglei-Cong) [inline consensus/vbft/service.go]: unnecessary change

**Review by Honglei-Cong — CHANGES_REQUESTED** [review]

File: consensus/vbft/chain_store.go (hunk 1 above)
Comment (xiemylogos) [reply consensus/vbft/chain_store.go]: ok

**Review by Honglei-Cong — APPROVED** [review]
